actors/verifreg/src/testing.rs

```diff
@@ -47,35 +44,5 @@ pub fn check_state_invariants<BS: Blockstore>(
         Err(e) => acc.add(format!("error loading verifiers {e}")),
     }
 
-    // check clients
-    let mut all_clients = HashMap::new();
-    match Map::<_, BigIntDe>::load(&state.verified_clients, store) {
-        Ok(clients) => {
-            let ret = clients.for_each(|key, cap| {
-                let client = Address::from_bytes(key)?;
-                let cap = &cap.0;
-
-                acc.require(
-                    client.protocol() == Protocol::ID,
-                    format!("client {client} should have ID protocol"),
-                );
-                acc.require(!cap.is_negative(), format!("client {client} cap {cap} is negative"));
-                all_clients.insert(client, cap.clone());
-                Ok(())
-            });
-
-            acc.require_no_error(ret, "error iterating clients");
-        }
-        Err(e) => acc.add(format!("error loading clients {e}")),
-    }
-
-    // check verifiers and clients are disjoint
```

Comment:
TODO: I think we should still check this since we appear to be trying to maintain the invariant. It will become a cross DataCap Verifreg check

Reply:
Yes, #537 is for that.